frontend/src/features/public-form/components/PublicSwitchEnvMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Text } from '@chakra-ui/react'
+
+import { UiCookieValues } from '~shared/types'
+
+import { publicChooseEnvironment } from '~services/EnvService'
+import InlineMessage from '~components/InlineMessage'
+import Link from '~components/Link'
+
+export const PublicSwitchEnvMessage = (): JSX.Element => {
+  const switchToUi = UiCookieValues.Angular
+
+  const handleClick = async () => {
+    await publicChooseEnvironment(switchToUi)
```

Comment:
should we use `react-query` mutations for this? adds a bunch of safety and loading states. check out our various `mutations.ts` files to see examples

```tsx
const switchEnvMutation = useMutation(
  () => publicChooseEnvironment(UiCookieValues.Angular), 
  { onSuccess: () => window.location.reload() }
)
```

Reply:
added `react-query` mutations!